esphome/components/touchscreen/touchscreen.cpp

```diff
@@ -13,26 +13,38 @@ void Touchscreen::attach_interrupt_(InternalGPIOPin *irq_pin, esphome::gpio::Int
   irq_pin->attach_interrupt(TouchscreenInterrupt::gpio_intr, &this->store_, type);
   this->store_.init = true;
   this->store_.touched = false;
+  ESP_LOGD(TAG, "Attach Touch Interupt");
+}
+
+void Touchscreen::call_setup() {
+  if (this->display_ != nullptr) {
+    this->display_width_ = this->display_->get_native_width();
+    this->display_height_ = this->display_->get_native_height();
+  }
+  PollingComponent::call_setup();
 }
 
 void Touchscreen::update() {
   if (!this->store_.init) {
     this->store_.touched = true;
   } else {
     // no need to poll if we have interrupts.
+    ESP_LOGW(TAG, "Touch Polling Stopped !!!");
```

Comment:
```suggestion
    ESP_LOGW(TAG, "Touch Polling Stopped. You can safely remove the "update_interval:" variable from the YAML file.");
```
You could see this as a yaml configuration problem where the user should have remove the `update_interval:` variable.